backend/src/tasks/s3-client.ts

```diff
@@ -54,6 +55,79 @@ class S3Client {
       throw e;
     }
   }
+
+  /**
+   * P&E Reports upload. Saves as a pdf in S3, then returns
+   * a temporary URL that can be used to access it.
+   */
+  async uploadPeReport(orgName: string, fileName: string) {
+    const Key = `pe-reports/${orgName}/${fileName}`;
+    const params = {
+      Bucket: process.env.EXPORT_BUCKET_NAME!,
+      Key,
+      Expires: 60 * 5, // 5 minutes
+      ContentType: 'application/pdf'
+    };
+    try {
+      const url = await this.s3.getSignedUrlPromise('putObject', params);
+      // Do this so exports are accessible when running locally.
+      if (this.isLocal) {
+        return url.replace('minio:9000', 'localhost:9000');
+      }
+      return url;
+    } catch (error) {
+      console.error('Error while generating presigned URL:', error.message);
+      throw new Error(error);
+    }
+  }
+  async exportPeReport(Key: string) {
+    try {
+      try {
+        await this.s3
+          .headObject({ Bucket: process.env.EXPORT_BUCKET_NAME!, Key })
```

Comment:
No need to call headObject, because if the file doesn't exist, then getSignedUrlPromise will just throw an error.

Reply:
Removed in https://github.com/cisagov/crossfeed/pull/1480/commits/d43cdb3f2812435112454e5e7fb8007d8f28bc28